src/uwtools/tests/fixtures/simple.ini

```diff
@@ -4,4 +4,3 @@ fruit = banana
 vegetable = tomato
 how_many = 12
 dressing = balsamic
-
```

Comment:
The dump logic in `INIConfig` no longer emits an extra trailing newline, so the fixture files were updated.